ansible/roles/miniflux/defaults/main.yml

```diff
@@ -253,9 +245,7 @@ miniflux__combined_configuration: '{{ miniflux__default_configuration
 
 miniflux__keyring__dependent_apt_keys:
 
-  - id: '{{ miniflux__upstream_gpg_key }}'
-    url: 'https://apt.miniflux.app/KEY.gpg'
-    repo: 'deb https://apt.miniflux.app/ /'
+  - repo: 'deb [trusted=yes] https://repo.miniflux.app/apt/ /'
```

Comment:
As I also mentioned for the variable name, the APT repository URL changed. Here is the [documentation](https://miniflux.app/docs/debian.html) for that.